main.go

```diff
@@ -10,13 +10,46 @@ import (
 	"github.com/opencontrol/fedramp-templater/templater"
 )
 
-func parseArgs() (openControlsDir, inputPath, outputPath string) {
-	if len(os.Args) != 4 {
-		log.Fatal("Usage:\n\n\tfedramp-templater <openControlsDir> <inputDoc> <outputDoc>\n\n")
+type options struct {
+	openControlsDir string
+	inputPath string
+	outputPath string
+	diff bool
+	fill bool
+}
+
+func printUsage() {
+	log.Fatal("Usage:\n\n" +
+		"\tfedramp-templater fill <openControlsDir> <inputDoc> <outputDoc>\n\n" +
+		"\tor\n\n" +
+		"\tfedramp-templater diff <openControlsDir> <inputDoc>")
```

Comment:
Minor: how about using [raw string literaral](https://golang.org/ref/spec#String_literals) (i.e. a multi-line string with backticks) so you don't need all of the `\n`s?